description.md

```diff
@@ -21,6 +21,41 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Tutorials
 
+Tutorials are pieces of content directed toward the reproduction or verification of a practical result.
+Tutorials are step-by-step presentations of a topic, and are supposed to be read by participants individually.
+They can be used as part of a practical session, with assistance provided to further explaining certain items, or as unassisted self-study.
+
+Tutorials consist of:
+
+- text: created following the contents of the [`Reading` section](#reading)
+- media: created following the contents of the [`Media` section](#section)
+- practice files: source code, diagrams or any digital information to be used by participants
+
+A participant would read the text, view media files and use (hands-on) practice files to get to a practical result.
+The tutorial presents content orderly, with the participant following instructions step-by-step.
+
+TODO: sample tutorials
+
+#### Tutorial Directory
+
+- contents.md
+- media/
+- support/ - source code, diagram, digital support files
```

Comment:
```suggestion
- support/ - files to be used when doing the tutorial: source code, diagram, digital support file
```

Reply:
Also, you are calling the directory support but in the taxonomy above you call this category "practice files". Choose one and stick to it.